lib/resty/limit/conn.lua

```diff
@@ -23,6 +23,29 @@ local mt = {
 }
 
 
+local function incr_or_init(key, dict)
```

Comment:
Would it make sense to ship this PR and then move to the new shdict API once https://github.com/openresty/lua-nginx-module/pull/579 is 🚢 'd?